api/specs/web-server/_folders.py

```diff
@@ -51,6 +48,8 @@ async def create_folder(_body: CreateFolderBodyParams):
 )
 async def list_folders(
     params: Annotated[PageQueryParameters, Depends()],
+    folder_id: FolderID | None = None,
+    workspace_id: WorkspaceID | None = None,
```

Comment:
this is for listing inside a folder or inside a workspace?

Reply:
Both!
- you can list folders inside a specific workspace (if workspace not provided then you are listing folders inside your private workspace)
- by default you list only folders in root, but if you provide specific folder id, we list folders inside that folder.